src/goto-programs/elf_reader.cpp

```diff
@@ -14,6 +14,39 @@ Module: Read ELF
 
 #include <istream>
 
+static void u16_to_native_endian_inplace(bool le_input, uint16_t &input)
```

Comment:
There is no variant of these in the standard library?

Reply:
There is `htons`, but it's not in the standard library and it doesn't fully cover what is needed here!